### PR TITLE
Fix for external-option being directly behind other options

### PR DESCRIPTION
**Pull request by rejas**
Noticed it when using the new proxy option. Also updated the changelog

### CODE REVIEW

File: analyze.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ function buildCommand(options){
     command += ' --proxy ' + options.proxy;
 
   if(options.noExternals)
-    command += '--no-externals'
+    command += ' --no-externals';
```
Comment (AvraamMavridis) [inline analyze.js:28]: good catch!

Comment (rejas) [reply analyze.js:28]: could you publish a new release for that fix (pretty please :-) ?

Comment (AvraamMavridis) [reply analyze.js:28]: yep, give me 10 mins and I will do it.

Comment (AvraamMavridis) [reply analyze.js:28]: Done! 2.4.0 on npm

Comment (AvraamMavridis) [reply analyze.js:28]: Thx for the PR

**Pull request merged**